sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/jms/ServiceBusJmsAutoConfiguration.java

```diff
@@ -60,17 +63,16 @@ AzureServiceBusJmsProperties serviceBusJmsProperties(AzureGlobalProperties azure
 
     @Bean
     @ConditionalOnExpression("'premium'.equalsIgnoreCase('${spring.jms.servicebus.pricing-tier}')")
-    ServiceBusJmsConnectionFactoryCustomizer amqpOpenPropertiesCustomizer(ObjectProvider<AzureServiceBusJmsCredentialSupplier> azureServiceBusJmsCredentialSupplier) {
+    AzureServiceBusJmsConnectionFactoryCustomizer amqpOpenPropertiesCustomizer(ObjectProvider<AzureServiceBusJmsCredentialSupplier> azureServiceBusJmsCredentialSupplier) {
         return factory -> {
             final Map<String, Object> properties = new HashMap<>();
-            properties.put("com.microsoft:is-client-provider", true);
             if (azureServiceBusJmsCredentialSupplier.getIfAvailable() != null) {
                 properties.put("user-agent", AZURE_SPRING_PASSWORDLESS_SERVICE_BUS);
             } else {
                 properties.put("user-agent", AZURE_SPRING_SERVICE_BUS);
             }
-            //set user agent
-            factory.setExtension(JmsConnectionExtensions.AMQP_OPEN_PROPERTIES.toString(),
+            JmsConnectionFactory jmsFactory = (JmsConnectionFactory) ReflectionUtils.getField(ServiceBusJmsConnectionFactory.class, "factory", factory);
+            jmsFactory.setExtension(JmsConnectionExtensions.AMQP_OPEN_PROPERTIES.toString(),
```

Comment:
This is not correct, since the sdk has already set this property in their implementation. The only thing we can do right now is to ignore our useragent.